### PR TITLE
2.0/feature navigation system/backend

### PR DESCRIPTION
**Pull request by maks100ful**

### CODE REVIEW

File: SportHub/src/modules/NavigationAdminSystem/components/NavigationItems/components/Dropdown/styles/style.module.scss
Comment (XashDM) [inline SportHub/src/modules/NavigationAdminSystem/components/NavigationItems/components/Dropdown/styles/style.module.scss:1]: Undo changes here

File: SportHub/src/modules/NavigationAdminSystem/components/NavigationItems/index.js
Comment (XashDM) [inline SportHub/src/modules/NavigationAdminSystem/components/NavigationItems/index.js:1]: Undo changes here

**Review by Oleh-Kril — COMMENTED** [review]
Merge master

**Review by Oleh-Kril — APPROVED** [review]
Approve because empty lines are minor and it's hard to rollback them

**Pull request merged**